sound/hybrid_font.h

```diff
@@ -230,18 +230,72 @@ class HybridFont : public SaberBase, public Looper {
     if (loop) hum_player_->PlayLoop(loop);
   }
 
+  int saved_player_to_restart_ = -1 ;
```

Comment:
I don't think this saved player to restart is the right way to do it.
In fact, it could potentially cause bugs since this player could be playing something completely different by the time we get around to try  to use it.

Either we should do the simplest possible thing, which is just to kill effects of the same type as the one we're now trying to play....

OR

We should do the full-on-complicated thing which, which means looking for any effect which is being played by multiple players right now, rank them by time left to play and kill the one with the smallest number, but that also means making sure that we don't hit vital effects, like effects with ref counts, looping effects, or an effect with a next pointer.